10-query/01b-pattern-negation.md

```diff
@@ -143,17 +143,161 @@ We shall now see how we can form more complex patterns with negation. Let's say
+
+```
+A = P \ F \ M = {Charlie}
+```
+
+Now let's complicate things a little and see what happens if we bind the `$y` variable for parents, i. e. if we consider a query pattern:
+
+```
+$x isa person;
+$y isa person;
 not { ($x, father: $y) isa parentship;};
 not { ($x, mother: $y) isa parentship;};
 ```
 
-This tells us to compute a set of answers `A = P \ F \ M`, where the set `P` is the set of all people, the set `F` is the set of all people having a father and the set `M` is the
-set of all people having a mother. Please not that the scope of variables in a negation block is local to the negation block. As a result the above pattern does not 
-look for people that do not have a mother and a father that is the same person.
+Upon inspection, we can see that in this case our three sets are different. Let's define them as `P'`, `M'` amd `F'`.
+The first difference is that an element of each set is a pair (2-tuple). Our set of people `P` corresponds to the answer set of a match query:
+
+```
+match
+$x isa person;
+$y isa person;
+get;
+```
+
+As a result, we have (abbreviating the names for clarity):
+
+```
+P' = { 
+        (A, A), (A, B), (A, C),
+        (B, A), (B, B), (B, C),
+        (C, A), (C, B), (C, C)
+}
+```
+
+Now the set `F'` is a set of pairs `{($x, $y)}` such that the concepts in the pair are connected via `parentship` relation with the concept assigned to `$y` variable playing the role of a father. 
+Similarly the set `M'` is a set of pairs `{($x, $y)}` such that the concepts in the pair are connected via `parentship` relation with the concept assigned to `$y` variable playing the role of a mother. 
+Consequently we have:
+
+```
+F' = {(A, B)}
+M' = {(B, C)}
+```
+
+Now, executing our query pattern as an ordinary match-get query:
+
+```
+match
+$x isa person;
+$y isa person;
+not { ($x, father: $y) isa parentship;};
+not { ($x, mother: $y) isa parentship;};
+get;
+```
+
+we yield the following concept pairs mapped to `$x` and `$y` variables:
+
+```
+A'' = P' \ F' \ M' = 
+{ 
+        (A, A), (A, C),
+        (B, A), (B, B), 
+        (C, A), (C, B), (C, C)
+}
+```
+
+However! If we now do a projection onto thr `$x` variable to compare the results with our previous query variant where 
+the `$y` variable is unbounded, i.e. if we execute:
+
+```
+match
+$x isa person;
+not { ($x, father: $y) isa parentship;};
+not { ($x, mother: $y) isa parentship;};
+get $x;
+```
+
+we will get the following concepts in return:
+
+```
+A' = P' \ F' \ M' | x = {A, B, C}
+```
+which is clearly different to the anticipated result of `A = {C}` - our answer set is not a set of orphans, instead
+it is a projection from a set of people pairs with pairs playing in `parentship` relation excluded. 
+As a result, extra care should be taken and thought given when formulating queries with negation blocks.
 
 One might be tempted to put the two negation blocks into one. Let's look at the outcome of that. If we define:
 
```

Comment:
Below (I can't comment on the correct line, line 312):

_From all the people,_ 

change to either:
 _From the set of all people,_
 _From all people,_